.github/workflows/check_dependency_duplications.yml

```diff
@@ -13,11 +13,11 @@ jobs:
   units:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-      - name: Set up Node.js 16
-        uses: actions/setup-node@v2.5.1
+      - uses: actions/checkout@v4
+      - name: Set up Node.js 20
```

Comment:
Let's remove the node version from name, so it will be less work to do in the future 😎